catalog/core/catalog-core-solrcommands/src/main/java/org/codice/ddf/commands/solr/SolrCommands.java

```diff
@@ -76,4 +83,32 @@ protected void printErrorMessage(String message) {
         printColor(ERROR_COLOR, message);
     }
 
+    protected void printInfoMessage(String message) {
+        printColor(INFO_COLOR, message);
+    }
+
+    private SolrClient getCloudSolrClient() {
+        String zkHosts = System.getProperty(ZOOKEEPER_HOSTS_PROP);
+        LOGGER.debug("Zookeeper hosts: {}", zkHosts);
+
+        if (zkHosts != null) {
+            SolrClient client = new CloudSolrClient.Builder().withZkHost(zkHosts)
+                    .build();
+            LOGGER.debug("Created solr client: {}", client);
+            return client;
+        } else {
+            return null;
+        }
+    }
+
+    void shutdown(SolrClient client) throws IOException {
```

Comment:
❓If `shutdown` and `getSolrClient` are meant to be used by sub-classes, should we make them `protected`?